WordPress/Tests/WP/DeprecatedParameterValuesUnitTest.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
```

Comment:
I'm missing unit tests testing the conditions on line 176 and 180.

Similarly, maybe add one unit test where one of the deprecated values is used in another position in the function call ?

Reply:
Still missing a unit test with a non-deprecated value (https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1366#discussion_r192925928):
`get_bloginfo( 'wpurl' ); // OK.`